store/todos.js

```diff
@@ -1,6 +1,11 @@
-import uuid4 from 'uuid4'
+import { createClient } from '@supabase/supabase-js'
 import { ALL, COMPLETE, INCOMPLETE } from '@/utils/constants.js'
 
+const supabase = createClient(
+  'https://zhsepptcavyowohakieh.supabase.co',
+  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Inpoc2VwcHRjYXZ5b3dvaGFraWVoIiwicm9sZSI6ImFub24iLCJpYXQiOjE2NzU0OTE3NzEsImV4cCI6MTk5MTA2Nzc3MX0.SxdLsTGvb6mtyTtqf7zpBkt7IxAVuHvl3r7b3DGVLH8'
+)
```

Comment:
We can create a Singleton class so that supabase instance isn't created multiple times.

Reply:
Resolved, Bhai. Thanks